src/monitoring_service/schema.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE blockchain (
+    chain_id                        INTEGER,
+    receiver                        CHAR(42),
+    token_network_registry_address  CHAR(42),
+    monitor_contract_address        CHAR(42),
+    latest_known_block              INT
+);
+INSERT INTO blockchain VALUES (
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    NULL
+);
+
+CREATE TABLE token_network (
+    address                 CHAR(42) PRIMARY KEY
+);
+
+
+CREATE TABLE channel (
+    token_network_address   CHAR(42) NOT NULL,
```

Comment:
Would it make sense to reference the `token_network` table here? I don't see any advantage at the moment because they are never removed.

Reply:
Yes, since we have the strict ordering here (as opposed to the monitoring request -> channel relation), it makes sense to add a foreign key. I'll do so.